measures/comstock_sensitivity_reports/measure.rb

```diff
@@ -2460,6 +2486,39 @@ def run(runner, user_arguments)
     end
     runner.registerValue('com_report_hvac_dx_heating_fraction_electric_defrost', dx_heating_fraction_electric_defrost)
 
+    # cycling ratio for unitary system
+    # initialize variables
+    com_report_unitary_sys_cycling_ratio_cooling_weighted_sum = 0.0
+    com_report_unitary_sys_cycling_ratio_heating_weighted_sum = 0.0
+    total_floor_area = 0.0
+    # loop through airloophvac unitary systems
+    model.getAirLoopHVACUnitarySystems.each do |airloopunisys|
+      # get timeseries data
+      ts_unitary_sys_cycling_ratio = sql.timeSeries(ann_env_pd, timeseries_timestep, 'Unitary System DX Coil Cycling Ratio', airloopunisys.name.to_s.upcase) # dimensionless
+      ts_unitary_sys_cycling_ratio = convert_timeseries_to_list(ts_unitary_sys_cycling_ratio)
+      ts_unitary_sys_sen_cooling_rate = sql.timeSeries(ann_env_pd, timeseries_timestep, 'Unitary System Sensible Cooling Rate', airloopunisys.name.to_s.upcase) # W
+      ts_unitary_sys_sen_cooling_rate = convert_timeseries_to_list(ts_unitary_sys_sen_cooling_rate)
+      ts_unitary_sys_sen_heating_rate = sql.timeSeries(ann_env_pd, timeseries_timestep, 'Unitary System Sensible Heating Rate', airloopunisys.name.to_s.upcase) # W
+      ts_unitary_sys_sen_heating_rate = convert_timeseries_to_list(ts_unitary_sys_sen_heating_rate)
+      # calculate average cycling ratio
+      ts_unitary_sys_cycling_ratio_cooling = ts_unitary_sys_cycling_ratio.zip(ts_unitary_sys_sen_cooling_rate).select { |a, b| b != 0 }.map { |a, b| a }
+      avg_unitary_sys_cycling_ratio_cooling = get_average_from_array(ts_unitary_sys_cycling_ratio_cooling)
+      ts_unitary_sys_cycling_ratio_heating = ts_unitary_sys_cycling_ratio.zip(ts_unitary_sys_sen_heating_rate).select { |a, b| b != 0 }.map { |a, b| a }
+      avg_unitary_sys_cycling_ratio_heating = get_average_from_array(ts_unitary_sys_cycling_ratio_heating)
+      # get floor area served
+      corresponding_thermal_zone = airloopunisys.thermalZone.get
+      corresponding_floor_area_m_2 = corresponding_thermal_zone.floorArea
```

Comment:
I don't know if floor area is best way to average. I think either by heating/cooling energy, or heating/cooling capacity would be better.

Reply:
yeah I can take a look on this one.

Reply:
[c5222de](https://github.com/NREL/ComStock/pull/190/commits/c5222de47ae1e13be5ffa51f08cbb77e05187292)